Object Storage/using-object-storage-from-3rd-party-tools.md

```diff
@@ -54,25 +54,31 @@ CenturyLink Cloud now has a robust, geo-distributed Object Storage service capab
 
-4. Also you will need the **Service Point** (aka server) where your bucket exists (or will exist). In the Control Portal, on the Object Storage bucket listing page, either create a new bucket, or click on an existing bucket. When the **Bucket Info and Settings** page is displayed, copy/save the portion of the **bucket URL** that follows the bucket name. In this example, the bucket **Service Point** (server name) is: **canada.os.ctl.io**.
+5. Also you will need the **Service Point** (aka server) where your bucket exists (or will exist). In the Control Portal, on the Object Storage bucket listing page, either create a new bucket, or click on an existing bucket. When the **Bucket Info and Settings** page is displayed, copy/save the portion of the **bucket URL** that follows the bucket name. In this example, the bucket **Service Point** (server name) is: **canada.os.ctl.io**.
    ![object storage bucket info](../images/object-storage-bucket-info-and-settings.png)
 
-5. Back in CyberDuck, click the **Open Connection** icon (upper left corner) in order to configure the connection details for your Object Storage bucket. Select **S3 (Amazon Simple Storage)** from the drop down. In the **Server** field, enter the name of the **Service point**. Leave Port field set to **443**. Populate the **Username** field with your **Access Key ID** and populate the **Password** field with your **Secret Access key**.
+6. Open the **AWS S3 AWS2 Signature Version Profile for HTTPS**.  In the **Server** field, enter the name of the **Service point**. Change the Port field to **443**. Populate the **Username** field with your **Access Key ID**. Close the window. This will create a bookmark in your **Bookmarks** list.
+
+7. Open the connection by clicking on the bookmark.  Populate the **Password** field with your **Secret Access key**.  Then click **Login**.
+
+  1. For creating connections later, replace steps the previous 2 steps with this: Click the **Open Connection** icon (upper left corner) in order to configure the connection details for your Object Storage bucket. Select **S3 (Amazon Simple Storage)** from the drop down. In the **Server** field, enter the name of the **Service point**. Leave Port field set to **443**. Populate the **Username** field with your **Access Key ID** and populate the **Password** field with your **Secret Access key**.
    ![cyberduck setup connection](../images/cyberduck-setup-connection.png)
 
-6. After you have configured the Object Storage connection, CyberDuck will list the available buckets at that **Connection**. You can double-click on a bucket to upload files or even drag-and-drop files into the bucket.
+8. After you have configured the Object Storage connection, CyberDuck will list the available buckets at that **Connection**. You can double-click on a bucket to upload files or even drag-and-drop files into the bucket.
    ![cyberduck established connection](../images/cyberduck-established-connection.png)
```

Comment:
check the preview as this isn't wrapping properly.  best to add a full line between the text and the image